src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -546,66 +580,65 @@ string Visitor::visitTypesafe(Parser::PnameNsContext* ctx) {
 }
 
 // ____________________________________________________________________________________
-SparqlQleverVisitor::PrefixMap SparqlQleverVisitor::visitTypesafe(
+SparqlQleverVisitor::PrefixMap SparqlQleverVisitor::visit(
     SparqlAutomaticParser::PrologueContext* ctx) {
   if (!ctx->baseDecl().empty()) {
     reportError(ctx->baseDecl(0), "BaseDecl is not supported.");
   }
   for (const auto& prefix : ctx->prefixDecl()) {
-    visitTypesafe(prefix);
+    visit(prefix);
   }
   // TODO: we return a part of our internal state here. This will go away when
   //  queries can be parsed completely with ANTLR.
   return _prefixMap;
 }
 
 // ____________________________________________________________________________________
-SparqlPrefix SparqlQleverVisitor::visitTypesafe(
+SparqlPrefix SparqlQleverVisitor::visit(
     SparqlAutomaticParser::BaseDeclContext* ctx) {
   reportError(ctx, "BaseDecl is not supported.");
 }
 
 // ____________________________________________________________________________________
-SparqlPrefix SparqlQleverVisitor::visitTypesafe(
+SparqlPrefix SparqlQleverVisitor::visit(
     SparqlAutomaticParser::PrefixDeclContext* ctx) {
   auto text = ctx->PNAME_NS()->getText();
   // Remove the ':' at the end of the PNAME_NS
   auto prefixLabel = text.substr(0, text.length() - 1);
-  auto prefixIri = visitTypesafe(ctx->iriref());
+  auto prefixIri = visit(ctx->iriref());
   _prefixMap[prefixLabel] = prefixIri;
   return {prefixLabel, prefixIri};
```

Comment:
This function should probably also return `void` by now.

Reply:
These are still require for testing. I'll look into how to rewrite the corresponding test in a nice way.